spectroscopy/code_src/spitzer_functions.py

```diff
@@ -32,71 +27,81 @@ def SpitzerIRS_get_spec(sample_table, search_radius_arcsec , COMBINESPEC):
     -------
     df_lc : MultiIndexDFObject
         The main data structure to store all spectra
-        
+
     '''
 
-    ## Initialize multi-index object:
+    # Initialize multi-index object:
     df_spec = MultiIndexDFObject()
-    
+
     for stab in sample_table:
-    
+
         print("Processing source {}".format(stab["label"]))
-        
-        ## Do search
+
+        # Do search
         search_coords = stab["coord"]
-        tab = Irsa.query_region(coordinates=search_coords, catalog="irs_enhv211", spatial="Cone",radius=search_radius_arcsec * u.arcsec)
+        tab = Irsa.query_region(coordinates=search_coords, catalog="irs_enhv211",
+                                spatial="Cone", radius=search_radius_arcsec * u.arcsec)
         print("Number of entries found: {}".format(len(tab)))
-    
-        if len(tab) > 0: # found a source
-        
-            ## If multiple entries are found, pick the closest.
+
+        if len(tab) > 0:  # found a source
+
+            # If multiple entries are found, pick the closest.
             # Or should we take the average instead??
             if len(tab) > 0:
-                print("More than 1 entry found" , end="")
+                print("More than 1 entry found", end="")
                 if not COMBINESPEC:
                     print(" - pick the closest")
```

Comment:
Beyond scope of this PR, but I noticed this:
in the lightcurves notebook/code we had a verbose option, so we don't get all the noise from these prints.

Reply:
Most of the other spectroscopy modules have the verbose option. Not sure why it was left out here.